backend/src/main/java/org/example/app/task/service/TaskService.java

```diff
@@ -0,0 +1,42 @@
+package org.example.app.task.service;
+
+import io.quarkus.security.Authenticated;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.media.Content;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.example.app.general.common.security.PermissionService;
+
+import static org.example.app.general.common.security.ApplicationAccessControlConfig.*;
+
+
+@Path("/task")
+@Authenticated
+public class TaskService {
```

Comment:
Is this code from testing? I guess it belongs to the solution branch and not on main.